src/raft/raft.rs

```diff
@@ -1165,10 +1165,10 @@ impl<T: Storage> Raft<T> {
         self.vote = hs.get_vote();
     }
 
-    // is_election_timeout returns true if self.election_elapsed is greater than the
-    // randomized election timeout in [electiontimeout, 2 * electiontimeout - 1].
-    // Otherwise, it returns false.
-    pub fn is_election_timeout(&self) -> bool {
+    /// `pass_election_timeout` returns true iff `election_elapsed` is greater
```

Comment:
The origin comments in etcd is "if", see https://github.com/coreos/etcd/commit/5d431b47822a579dc9fb88342f3a1dfa7f160d11#diff-a8629a0eb6da1c6c3805111f1f1af85eR872
Please keep them sync.